packages/cli/src/commands/dev/runDevServer.js

```diff
@@ -26,7 +26,9 @@ async function runDevServer({ context }) {
       cwd: context.directories.devServer,
       env: {
         ...process.env,
+        LOWDEFY_PACKAGE_MANAGER: context.packageManager,
         LOWDEFY_DIRECTORY_CONFIG: context.directories.config,
+        PORT: context.options.port,
```

Comment:
Sjouldn't this rather be something like `LOWDEFY_PORT` or `LOWDEFY_SERVER_PORT`?

Reply:
Nextjs looks at PORT. https://nextjs.org/docs/api-reference/cli#development
So if we set PORT, then the Next server is configured. Think we should keep it the same?